src/midiEditorCommands.cpp

```diff
@@ -455,6 +484,9 @@ void cmdMidiMovePitchCursor(Command* command) {
 	int chan = MIDIEditor_GetSetting_int(editor, "default_note_chan");
 	int vel = MIDIEditor_GetSetting_int(editor, "default_note_vel");
 	previewNotes(take, {{chan, pitch, vel}});
+	if (!shouldReportNotes) {
+		return;
```

Comment:
nit: I'm a big fan of early returns where it makes things simpler, but in this case, I think it's more readable to reverse this and put the outputMessage inside the if block.